packages/common/src/utils/wallet.ts

```diff
@@ -110,18 +111,27 @@ export const formatWei = (
   return formatNumberCommas(trimmed) as StringAudio
 }
 
-export const convertJSBIToAmountObject = (amount: JSBI, decimals: number) => {
-  const divisor = JSBI.BigInt(10 ** decimals)
-  const quotient = JSBI.divide(amount, divisor)
-  const remainder = JSBI.remainder(amount, divisor)
-  const uiAmountString = JSBI.greaterThan(remainder, JSBI.BigInt(0))
-    ? `${quotient.toString()}.${remainder.toString().padStart(decimals, '0')}`
-    : quotient.toString()
+export const convertQuoteStringToAmountObject = (
+  amount: string,
+  decimals: number
+): AmountObject => {
   return {
-    amount: JSBI.toNumber(amount),
+    amount: Number(BigInt(amount)),
+    amountString: amount,
+    uiAmount: Number(BigInt(amount)) / 10 ** decimals,
+    uiAmountString: (BigInt(amount) / BigInt(10 ** decimals)).toString()
+  }
+}
+
+export const convertBigIntToAmountObject = (
+  amount: bigint,
+  decimals: number
+): AmountObject => {
+  return {
+    amount: Number(amount),
     amountString: amount.toString(),
-    uiAmount: JSBI.toNumber(amount) / 10 ** decimals,
-    uiAmountString
+    uiAmount: Number(amount) / 10 ** decimals,
+    uiAmountString: (Number(amount) / 10 ** decimals).toString()
```

Comment:
```(Number(123456789012345678901234567890n) / 10 ** 6).toString()```
is `'1.2345678901234569e+23'`

lets do string manipulation instead. something like:
```
const bign = 123456789012345678901234567890n
const str = bign.toString()
const decimals = 6
const res = `${str.substring(0, str.length - decimals)}.${str.substring(str.length - decimals)}`


// res =  '123456789012345678901234.567890'
```

Alternatively, [this is what I do for JSBI conversion](https://github.com/AudiusProject/audius-protocol/blob/36eece925d954b16c6b8c5f070a6684b254c2ecf/packages/common/src/utils/wallet.ts#L117-L119) to amount object, which we could do here:

```
const bign = -123456789012345678901234567890n
const decimals = 6
const divisor = BigInt(10 ** decimals)
const quotient = bign / divisor
const remainder = quotient > 0 ? bign % divisor : -1n * (bign % divisor)
const res = `${quotient}.${remainder.toString().padStart(decimals, '0')}`

// res = '-123456789012345678901234.567890'
```